README.md

```diff
@@ -24,24 +24,97 @@ require 'opal/rspec/rake_task'
 Opal::RSpec::RakeTask.new(:default)
 ```
 
-Then, to run your specs inside phantomjs, just run the rake task:
+Then, to run your specs inside phantomjs (the default runner), just run the rake task:
 
 ```
 bundle exec rake
 ```
 
+Enable colors in the output
+
+```
+SPEC_OPTS="--color" bundle exec rake
+```
+
+Use a different formatter
+
+```
+SPEC_OPTS="--format json" bundle exec rake
+```
+
+The following formatters have been tested:
+* Default (progress)
+* Documentation
+* JSON
+
+If you need to specify additional requires for a custom formatter, you can do this:
+
+```
+SPEC_OPTS="--format SomeFormatter --require some_formatter" bundle exec rake
+```
+
+You can also customize the pattern of specs used similiar to how RSpec's rake task works:
+
+```ruby
+require 'opal/rspec/rake_task'
+Opal::RSpec::RakeTask.new(:default) do |server, task|
+	# server is an instance of Opal::Server in case you want to add to the load path, customize, etc.
+	task.pattern = 'spec_alternate/**/*_spec.rb' # can also supply an array of patterns
```

Comment:
@wied03 can you convert all tabs to two spaces from the README and ?

(found some in `README.md` and `spec/mri/integration/rack/sprockets_runner_js_errors.rb.erb`)